WordPressKit/RemoteReaderSiteInfo.m

```diff
@@ -50,9 +49,7 @@ + (instancetype)siteInfoForSiteResponse:(NSDictionary *)response isFeed:(BOOL)is
     siteInfo.siteName = [response stringForKey:SiteDictionaryNameKey];
     siteInfo.siteURL = [response stringForKey:SiteDictionaryURLKey];
     siteInfo.subscriberCount = [response numberForKey:SiteDictionarySubscriptionsKey] ?: @0;
-
-    NSDictionary *options = [response valueForKey:SiteDictionaryOptionsKey];
-    siteInfo.isWPForTeams = [[options numberForKey:SiteDictionaryIsWPForTeamsKey] boolValue];
+    siteInfo.organizationID = [response numberForKey:SiteDictionaryOrganizationID];
```

Comment:
Should this include a ` ?: @0;` ?

Reply:
Yep! Working on it right now. Sorry I didn't catch it before you reviewed.